scripts/solution_view.py

```diff
@@ -135,45 +157,92 @@ def pos_llh_callback(self, data):
       t = datetime.datetime(1980, 1, 6) + \
           datetime.timedelta(weeks=self.week) + \
           datetime.timedelta(seconds=tow)
-      self.pos_table.append(('GPS Time', t))
-      self.pos_table.append(('GPS Week', str(self.week)))
+      pos_table.append(('GPS Time', t))
+      pos_table.append(('GPS Week', str(self.week)))
 
-      self.log_file.write('%s,%.10f,%.10f,%.4f,%d\n' % (
+      self.log_file.write('%s,%.10f,%.10f,%.4f,%d,%d\n' % (
         str(t),
         soln.lat, soln.lon, soln.height,
-        soln.n_sats)
+        soln.n_sats,soln.flags)
       )
       self.log_file.flush()
 
-    self.pos_table.append(('GPS ToW', tow))
-
-    self.pos_table.append(('Num. sats', soln.n_sats))
-
-    self.pos_table.append(('Lat', soln.lat))
-    self.pos_table.append(('Lng', soln.lon))
-    self.pos_table.append(('Alt', soln.height))
+    pos_table.append(('GPS ToW', tow))
+
+    pos_table.append(('Num. sats', soln.n_sats))
+
+    pos_table.append(('Lat', soln.lat))
+    pos_table.append(('Lng', soln.lon))
+    pos_table.append(('Alt', soln.height))
+    pos_table.append(('Flags', '0x%02x' % soln.flags))
+    if (soln.flags & 0xff) == 0:
+      pos_table.append(('Mode', 'SPP (single point position)'))
+    elif (soln.flags & 0xff) == 1:
+      pos_table.append(('Mode', 'Fixed RTK'))
+    elif (soln.flags & 0xff) == 2:
+      pos_table.append(('Mode', 'Float RTK'))
+    else:
+      pos_table.append(('Mode', 'Unknown'))
+
+    if psuedo_absolutes==True:
+      #setup_plot variables
+      self.lats_psuedo_abs.append(soln.lat)
+      self.lngs_psuedo_abs.append(soln.lon)
+      self.alts_psuedo_abs.append(soln.height)
+
+      self.lats_psuedo_abs = self.lats_psuedo_abs[-1000:]
+      self.lngs_psuedo_abs = self.lngs_psuedo_abs[-1000:]
+      self.alts_psuedo_abs = self.alts_psuedo_abs[-1000:]
+
+      self.plot_data.set_data('lat_ps', self.lats_psuedo_abs)
+      self.plot_data.set_data('lng_ps', self.lngs_psuedo_abs)
+      self.plot_data.set_data('alt_ps', self.alts_psuedo_abs)
+      self.plot_data.set_data('cur_lat_ps', [soln.lat])
+      self.plot_data.set_data('cur_lng_ps', [soln.lon])
+      t_psuedo_abs = range(len(self.lats))
+      self.plot_data.set_data('t', t)
+      self.plot_data.set_data('t_ps', t_psuedo_abs)
+      #set-up table variables
+      self.pos_table_psuedo_abs = pos_table
```

Comment:
Can probably get rid of `pos_table_psuedo_abs` seeing as this is the only place you set `table2`. For the SPP table we had to store another copy of just the position related items so that we could use them when updating just the DOPs and the velocities. Here I think you can just do `self.table2 = pos_table`